### ISSUE TITLE
Bad URL scheme on CLI causes crash

### ISSUE THREAD

**Issue by ry** [denoland/deno#2593]
```shell
$ deno info htts://deno.land/std/examples/catjson.ts
thread 'tokio-runtime-worker-0' panicked at 'not yet implemented', ../../cli/deno_dir.rs:335:12
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
thread 'main' panicked at 'internal error: entered unreachable code', ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/tokio-0.1.21/src/runtime/threadpool/mod.rs:296:26
thread 'tokio-runtime-worker-0' panicked at 'unexpected state while aborting task: Complete', ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/tokio-threadpool-0.1.14/src/task/mod.rs:216:21
stack backtrace:
   0:        0x108ea8573 - <std::sys::unix::backtrace::tracing::imp::UnwindError as core::fmt::Display>::fmt::h91279d169ccb5b2c
   1:        0x108ea11b2 - std::sys_common::alloc::realloc_fallback::h1ea3e7dd66f55875
   2:        0x108ea4b16 - std::panicking::take_hook::hf5878833b3146ef1
   3:        0x108ea4821 - std::panicking::take_hook::hf5878833b3146ef1
   4:        0x108ea528f - std::panicking::rust_panic_with_hook::h547ea459e27be908
   5:        0x108ea4dbc - std::panicking::begin_panic_fmt::hcacc0f82ca07e29f
   6:        0x108ea4d10 - std::panicking::begin_panic_fmt::hcacc0f82ca07e29f
   7:        0x109cb7186 - <tokio_threadpool::task::Task::run::{{closure}}::Guard<'a> as core::ops::drop::Drop>::drop::h3e4ab556eba4c96c
   8:        0x109cb1369 - tokio_threadpool::pool::state::<impl core::convert::From<tokio_threadpool::pool::state::Lifecycle> for usize>::from::h025a5dce3d6dcb6a
   9:        0x109cb8ece - <tokio_threadpool::shutdown::ShutdownTrigger as core::ops::drop::Drop>::drop::hefa5d5ec07d4e8ed
  10:        0x109cb26eb - <tokio_threadpool::worker::state::Lifecycle as core::fmt::Debug>::fmt::he88a1c5c815cc32d
  11:        0x109cbacf7 - <tokio_threadpool::pool::backup::Handoff as core::fmt::Debug>::fmt::h5e6953337ce61e91
  12:        0x109cb658f - <tokio_threadpool::pool::Pool as core::fmt::Debug>::fmt::h36088954a4cf63c5
  13:        0x108eb4f7e - __rust_maybe_catch_panic
  14:        0x109cad8c5 - <tokio_threadpool::thread_pool::Inner as core::fmt::Debug>::fmt::h17e9019c36f89f75
  15:        0x108eb3d8b - std::sys::unix::thread::Thread::new::h3263e5486f1e27cd
  16:     0x7fff717b42ea - _pthread_body
  17:     0x7fff717b7248 - _pthread_start
thread panicked while panicking. aborting.
Illegal instruction: 4
```

**Comment by bartlomieju** [comment]
Will be resolved by #2572

**Comment by bartlomieju** [comment]
Fixed in #2572 

```
$ deno_dev htts://deno.land/std/http/file_server.ts -A
Unsupported scheme "htts" for module "htts://deno.land/std/http/file_server.ts". Supported schemes: [
    "http",
    "https",
    "file",
]

```

We'll probably want to update the message, but it's not crashing anymore :)

CC @ry

**Comment by ry** [comment]
Actually it doesn't seem to be fixed.

```
~/src/deno> deno version
deno: 0.11.0
v8: 7.7.37
typescript: 3.5.1

~/src/deno> deno info htts://deno.land/std/http/file_server.ts
thread 'tokio-runtime-worker-0' panicked at 'not yet implemented', ../../cli/deno_dir.rs:335:12
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
thread 'main' panicked at 'internal error: entered unreachable code', ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/tokio-0.1.21/src/runtime/threadpool/mod.rs:296:26
thread 'main' panicked at 'unexpected state while aborting task: Complete', ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/tokio-threadpool-0.1.14/src/task/mod.rs:216:21
stack backtrace:
   0:        0x10c3bd573 - <std::sys::unix::backtrace::tracing::imp::UnwindError as core::fmt::Display>::fmt::hea0dae9c7c6e8ae9
   1:        0x10c3b5ad2 - std::sys_common::alloc::realloc_fallback::h9d38607bbcbe332d
   2:        0x10c3b9b16 - std::panicking::take_hook::hff7ae2469c321953
   3:        0x10c3b98bf - std::panicking::take_hook::hff7ae2469c321953
   4:        0x10c3ba28f - std::panicking::rust_panic_with_hook::hfcf2d0777bc6c409
   5:        0x10c3b9dbc - std::panicking::begin_panic_fmt::h1334cc5c63aa208c
   6:        0x10c3b9d10 - std::panicking::begin_panic_fmt::h1334cc5c63aa208c
   7:        0x10d1d0046 - <tokio_threadpool::task::Task::run::{{closure}}::Guard as core::ops::drop::Drop>::drop::h92c11a556351f63e
   8:        0x10d1c9c69 - tokio_threadpool::pool::state::<impl core::convert::From<tokio_threadpool::pool::state::Lifecycle> for usize>::from::h4674ed7dd15c5dcf
   9:        0x10d1d1d8e - <tokio_threadpool::shutdown::ShutdownTrigger as core::ops::drop::Drop>::drop::hae42e9b735eeedfd
  10:        0x10d1cc29b - <tokio_threadpool::worker::state::Lifecycle as core::fmt::Debug>::fmt::haca5ad760bc2c50e
  11:        0x10d1c4819 - tokio_threadpool::thread_pool::ThreadPool::shutdown_now::h3dea232b06e1e253
  12:        0x10d1a548c - tokio::runtime::current_thread::runtime::Runtime::run::hf065fc0530cd42f1
  13:        0x10d1a78b0 - <tokio::runtime::threadpool::Runtime as core::ops::drop::Drop>::drop::h85f3fb3f7dad8021
  14:        0x10d3e6b1b - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  15:        0x10d4f216c - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  16:        0x10d4f44c2 - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  17:        0x10d3d1fb5 - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  18:        0x10c3b9c27 - std::panicking::update_panic_count::h70e4b2e98402c826
  19:        0x10c3c972e - __rust_maybe_catch_panic
  20:        0x10c3ba70d - std::rt::lang_start_internal::h4c054360e442146c
  21:        0x10d4f79e8 - main
thread panicked while panicking. aborting.
Illegal instruction: 4

Error 132 ~/src/deno> deno  htts://deno.land/std/http/file_server.ts
thread 'tokio-runtime-worker-0' panicked at 'not yet implemented', ../../cli/deno_dir.rs:335:12
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
thread 'main' panicked at 'internal error: entered unreachable code', ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/tokio-0.1.21/src/runtime/threadpool/mod.rs:296:26
thread 'main' panicked at 'unexpected state while aborting task: Complete', ../../third_party/rust_crates/registry/src/github.com-1ecc6299db9ec823/tokio-threadpool-0.1.14/src/task/mod.rs:216:21
stack backtrace:
   0:        0x10851e573 - <std::sys::unix::backtrace::tracing::imp::UnwindError as core::fmt::Display>::fmt::hea0dae9c7c6e8ae9
   1:        0x108516ad2 - std::sys_common::alloc::realloc_fallback::h9d38607bbcbe332d
   2:        0x10851ab16 - std::panicking::take_hook::hff7ae2469c321953
   3:        0x10851a8bf - std::panicking::take_hook::hff7ae2469c321953
   4:        0x10851b28f - std::panicking::rust_panic_with_hook::hfcf2d0777bc6c409
   5:        0x10851adbc - std::panicking::begin_panic_fmt::h1334cc5c63aa208c
   6:        0x10851ad10 - std::panicking::begin_panic_fmt::h1334cc5c63aa208c
   7:        0x109331046 - <tokio_threadpool::task::Task::run::{{closure}}::Guard as core::ops::drop::Drop>::drop::h92c11a556351f63e
   8:        0x10932ac69 - tokio_threadpool::pool::state::<impl core::convert::From<tokio_threadpool::pool::state::Lifecycle> for usize>::from::h4674ed7dd15c5dcf
   9:        0x109332d8e - <tokio_threadpool::shutdown::ShutdownTrigger as core::ops::drop::Drop>::drop::hae42e9b735eeedfd
  10:        0x10932d29b - <tokio_threadpool::worker::state::Lifecycle as core::fmt::Debug>::fmt::haca5ad760bc2c50e
  11:        0x109325819 - tokio_threadpool::thread_pool::ThreadPool::shutdown_now::h3dea232b06e1e253
  12:        0x10930648c - tokio::runtime::current_thread::runtime::Runtime::run::hf065fc0530cd42f1
  13:        0x1093088b0 - <tokio::runtime::threadpool::Runtime as core::ops::drop::Drop>::drop::h85f3fb3f7dad8021
  14:        0x109546b4b - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  15:        0x10965462c - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  16:        0x109654c78 - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  17:        0x109532fb5 - arc_swap::debt::THREAD_HEAD::__getit::h4e2fe9dc7b30b9dd
  18:        0x10851ac27 - std::panicking::update_panic_count::h70e4b2e98402c826
  19:        0x10852a72e - __rust_maybe_catch_panic
  20:        0x10851b70d - std::rt::lang_start_internal::h4c054360e442146c
  21:        0x1096589e8 - main
thread panicked while panicking. aborting.
Illegal instruction: 4
```
